src/hyperlight_host/src/mem/layout.rs

```diff
@@ -604,11 +604,8 @@ impl SandboxMemoryLayout {
         }
 
         // guard page
-        let stack_offset = builder.push_page_aligned(
-            PAGE_SIZE_USIZE,
-            MemoryRegionFlags::READ | MemoryRegionFlags::STACK_GUARD,
-            GuardPage,
-        );
+        let stack_offset =
+            builder.push_page_aligned(PAGE_SIZE_USIZE, MemoryRegionFlags::READ, GuardPage);
```

Comment:
No more `STACK_GUARD`? Guess we don't need it w/ the `RegionType` being there...

Reply:
Yeah I commented earlier in #469

> It was not a real permissions flag like READ or WRITE or EXECUTE, it was only an indicator. But that makes no sense because we already have another indicator for whether a region is a guard page. I don't think flag serves any purpose, and only messes things up when coverting a memory region to a vm-specific region. For example it makes us unable to map a region with only READ (which means readonly) to for example KVM_MEM_READONLY, without special handling of STACK_GUARD, which I am not a big fan of.